.fuel-core/configs/chainConfig.json

```diff
@@ -98,168 +98,168 @@
               "base": 3,
-              "units_per_gas": 523
+              "units_per_gas": 564
             }
           },
           "mcli": {
             "LightOperation": {
               "base": 3,
-              "units_per_gas": 526
+              "units_per_gas": 560
             }
           },
           "mcp": {
             "LightOperation": {
-              "base": 3,
-              "units_per_gas": 448
+              "base": 4,
+              "units_per_gas": 185
             }
           },
           "mcpi": {
             "LightOperation": {
-              "base": 7,
-              "units_per_gas": 585
+              "base": 9,
+              "units_per_gas": 455
             }
           },
           "meq": {
             "LightOperation": {
-              "base": 11,
-              "units_per_gas": 1097
+              "base": 3,
+              "units_per_gas": 766
             }
           },
-          "retd_contract": {
+          "retd": {
             "LightOperation": {
-              "base": 1086,
-              "units_per_gas": 2
+              "base": 485,
+              "units_per_gas": 3
             }
           },
           "s256": {
             "LightOperation": {
-              "base": 45,
+              "base": 42,
               "units_per_gas": 3
             }
           },
           "scwq": {
             "HeavyOperation": {
-              "base": 30375,
-              "gas_per_unit": 28628
+              "base": 21672,
+              "gas_per_unit": 22146
             }
           },
           "smo": {
             "LightOperation": {
-              "base": 64196,
+              "base": 44437,
               "units_per_gas": 1
             }
           },
           "srwq": {
             "HeavyOperation": {
-              "base": 262,
-              "gas_per_unit": 249
+              "base": 239,
+              "gas_per_unit": 234
             }
           },
           "swwq": {
             "HeavyOperation": {
-              "base": 28484,
-              "gas_per_unit": 26613
+              "base": 22724,
+              "gas_per_unit": 21231
             }
           },
           "contract_root": {
             "LightOperation": {
-              "base": 45,
-              "units_per_gas": 1
+              "base": 42,
+              "units_per_gas": 2
             }
           },
           "state_root": {
             "HeavyOperation": {
-              "base": 350,
-              "gas_per_unit": 176
+              "base": 323,
+              "gas_per_unit": 169
             }
           },
           "new_storage_per_byte": 63,
           "vm_initialization": {
-            "LightOperation": {
-              "base": 1645,
-              "units_per_gas": 14
+            "HeavyOperation": {
+              "base": 5254820,
+              "gas_per_unit": 0
             }
           }
         }
       },
       "base_asset_id": "0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
@FuelLabs/sdk-ts Should we use the `base_asset_id` to the same ID used by the Testnet as well?